src/domain/registration/utils.ts

```diff
@@ -256,3 +257,55 @@ export const getRegistrationActionButtonProps = ({
     onClick,
   };
 };
+
+const downloadBlob = (blob: Blob, filename: string) => {
+  const blobUrl = window.URL.createObjectURL(blob);
+  const link = document.createElement('a');
+  link.href = blobUrl;
+  link.download = filename;
+  link.click();
+
+  // Release the object URL and remove the link
+  window.URL.revokeObjectURL(blobUrl);
+  link.remove();
+};
+
+export const exportSignupsAsExcel = async ({
+  addNotification,
+  registration,
+  session,
+  t,
+}: {
+  addNotification: (props: NotificationProps) => void;
+  registration: Registration;
+  session: ExtendedSession | null;
+  t: TFunction;
+}) => {
+  const url = `/registration/${registration.id}/signups/export/xlsx/`;
```

Comment:
The back-end has the GET parameter `ui_language` - that should probably be used here so that the XLSX would be translated according to the user's UI language.

Reply:
Added `ui_language` parameter to the reguest